### PR TITLE
Fix: Municipality selection on municipal index for mobile

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard1**
## Summary

The municipal map didn't show up properly on the municipal index. The tooltip has been made clickable as well, just like the escalation map.
Also, a type check error was fixed, so after this, the build should be working properly again.

## Motivation

Bug fix.

## Detailed design

Basically the same patterns were followed as the escalation map on the region index.

## Drawbacks

n/a

## Alternatives

n/a

## Unresolved questions

n/a

### Governance

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License
      https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt
      and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

### CODE REVIEW

File: src/pages/gemeente/index.tsx [hunk 1]
```diff
@@ -29,6 +40,7 @@ const tooltipContent = (context: MunicipalityProperties): ReactNode => {
 // lots of unnecessary null checks on those pages.
 const Municipality: FCWithLayout<any> = () => {
   const router = useRouter();
+  const isLargeScreen = useMediaQuery('(min-width: 1000px)');
```
Comment (VWSCoronaDashboard4) [inline src/pages/gemeente/index.tsx:43]: As an improvement later we could try and move these to CSS

Comment (VWSCoronaDashboard1) [reply src/pages/gemeente/index.tsx:43]: Please add a Trello ticket for these kinds of improvements

File: src/scss/layout.scss [hunk 2]
```diff
@@ -176,10 +176,13 @@
 
 @media (max-width: 999px) {
   .small-screen-aside-tendency {
-    .national-content,
-    .municipality-content {
+    .national-content {
```
Comment (VWSCoronaDashboard4) [inline src/scss/layout.scss:179]: This is not done well; the display none should be on the safety-region and municpality content too

Comment (VWSCoronaDashboard1) [reply src/scss/layout.scss:179]: but in that case the maps aren't visible on the indexes...

Comment (VWSCoronaDashboard4) [inline src/scss/layout.scss:179]: lines above should not be deleted ;  display none should be there for municpal and safety

**Review by VWSCoronaDashboard4 — APPROVED** [review]

File: src/scss/layout.scss (hunk 2 above)
Comment (VWSCoronaDashboard4) [reply src/scss/layout.scss:179]: my bad, intended menu improvements are intended :sunglasses:

**Pull request merged**